spec/vcr/Cloud9Invite/1_3.yml

```diff
@@ -0,0 +1,268 @@
+      - gzip, deflate
+      User-Agent:
+      - rest-client/2.0.0 (linux-gnu x86_64) ruby/2.3.1p112
+      Cookie:
+      - c9.live=<CLOUD9_LIVE_COOKIE>; c9.live.proxy=<CLOUD9_LIVE_PROXY_COOKIE>
+      Host:
+      - c9.io
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      X-Frame-Options:
+      - DENY
+      Set-Cookie:
+      - c9.live.proxy=<CLOUD9_LIVE_PROXY_COOKIE>; Domain=.c9.io; Path=/; Expires=Fri,
+        20 Jan 2017 23:45:17 GMT; HttpOnly
+      - c9.live=<CLOUD9_LIVE_COOKIE>; Path=/; Expires=Fri, 20 Jan 2017 23:45:17 GMT;
+        HttpOnly; Secure
+      Content-Type:
+      - text/plain
+      Access-Control-Allow-Origin:
+      - "*"
+      Access-Control-Allow-Methods:
+      - GET, OPTIONS, PUT, POST, DELETE, HEAD, PROPFIND, PATCH
+      Access-Control-Allow-Headers:
+      - Content-Type, X-Requested-With, x-request-metadata
+      Access-Control-Expose-Headers:
+      - x-content-length, x-metadata-length
+      Access-Control-Max-Age:
+      - '8640000'
+      Date:
+      - Fri, 13 Jan 2017 23:45:17 GMT
+      Transfer-Encoding:
+      - chunked
+      Content-Encoding:
+      - gzip
+      X-Backend:
+      - oldclient
+      Strict-Transport-Security:
+      - max-age=31536000
+    body:
+      encoding: ASCII-8BIT
+      string: !binary |-
+        H4sIAAAAAAAEA7NM9o2M8HVyKa20NHDKMSg2TstLBwAoH8lvFAAAAA==
+    http_version: 
+  recorded_at: Fri, 13 Jan 2017 23:45:17 GMT
+- request:
+    method: post
+    uri: https://api.c9.io/user/org/ENV-cloud9_team_name/invite?access_token=<CLOUD9_ACCESS_TOKEN>
+    body:
+      encoding: UTF-8
+      string: '{"email":null}'
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      User-Agent:
+      - rest-client/2.0.0 (linux-gnu x86_64) ruby/2.3.1p112
+      Content-Type:
+      - application/json
+      Content-Length:
+      - '14'
+      Host:
+      - api.c9.io
+  response:
+    status:
+      code: 422
+      message: Unprocessable Entity
+    headers:
+      Content-Type:
+      - application/json
+      Access-Control-Allow-Origin:
+      - "*"
+      Access-Control-Allow-Methods:
+      - GET, OPTIONS, PUT, POST, DELETE, HEAD, PROPFIND, PATCH
+      Access-Control-Allow-Headers:
+      - Content-Type, X-Requested-With, x-request-metadata
+      Access-Control-Expose-Headers:
+      - x-content-length, x-metadata-length
+      Access-Control-Max-Age:
+      - '8640000'
+      Date:
+      - Fri, 13 Jan 2017 23:45:17 GMT
+      Transfer-Encoding:
+      - chunked
+      X-Backend:
+      - api
+      Strict-Transport-Security:
+      - max-age=31536000
+    body:
+      encoding: UTF-8
+      string: '{"error":{"code":422,"hostname":"newclient-gce-usw-150-prod","scope":"api","message":"Validation
+        failed","errors":[{"resource":"root","field":"email","type_expected":"string","code":"invalid"}]}}'
+    http_version: 
+  recorded_at: Fri, 13 Jan 2017 23:45:18 GMT
+recorded_with: VCR 3.0.3
```

Comment:
I'm not sure if you meant to commit this file. It looks like this scenario only happens when you POST an invalid email to Cloud9, which we should never be doing.